api/updatepb/status.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package update;
+
+option go_package = "api/updatepb;updatepb";
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "google/api/annotations.proto";
+
+message AvailableRequest {}
+
+message AvailableResponse {}
+
+// Status service provides public status APIs for pmm-server-upgrade.
+service Status {
+  // Available checks if updater is available.
+  rpc Available(AvailableRequest) returns (AvailableResponse) {
+    option (google.api.http) = {
+      post: "/v1/Available"
```

Comment:
```suggestion
      post: "/v1/update/Status/Available"
```

Reply:
Generally I prefer for `ready`, `health`, `available`, etc. endpoints to be in root.
Any reason you'd prefer to nest it under `update/Status`?

Reply:
That's just rule that majority of existing endpoints follow. I guess better to follow same pattern everywhere.

P.S. checked sources, we have `/version/package/service/endpoint`, `/version/service/endpoint`, `/version/endpoint` patterns. Maybe it makes sense to discuss in slack with other devs. It's not clear for me what our rules regarding this.

Reply:
BTW if to use `/v1/Available` then it's not clear what is available?

Reply:
The API is specifically for `pmm-server-upgrade`. The endpoint shall return the state of the app as a whole.
I'll check in Slack to see what the general guideline preference is.

Reply:
To me the variant of `/version/endpoint` suffers from one major downside - all endpoints would end up in the root, providing no clue to which part of the app they refer.

For some of us the variant of `/v1/update/Status/Available`  may seem to be overly verbose. However, if we try to reduce it to just `/v1/update/Available` or `/v1/Status/Available`, it will suffer from the same problem as the shortest version above - lack of semantics and lack of origin (either package or service or both).

That said, I'm in favor of the long variant as the most semantic. Personally, the benefit is that I would have to refer less to the docs to understand what it does. 

I understand that, with a few exceptions, this is the convention we currently use.

Reply:
Gosh, I read method description twice and though that it checks that PMM server update are available. It it's just health check for `pmm-server-upgrade` service itself, then I think we should follow K8S naming rules here.
https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health

Like we do it in PMM: https://github.com/percona/pmm/blob/0b9ef90cae43e04490a61b858073967ff0116067/api/serverpb/server.proto#L310

Reply:
I removed the JSON rpc altogether. It wasn't used either way. It's not needed for our services to communicate with each other.